docs/source/developer/contributing.rst

```diff
@@ -204,6 +204,11 @@ well-defined coding style (would it be nice to follow :pep:`7`?).
 Code and documentation should generally fit within 80 columns, for
 maximum readability with all existing tools (such as code review UIs).
 
+In addition:
+
+* Docstrings should adhere to the [NumPy docstring convention](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html).
+* Lines are terminated with \r\n (Windows style).
```

Comment:
There is no need to standardize line terminations in the source code.  Git automatically translates between line endings when checking out on different platforms as long as you use the form that is standard on your platform.